db/db_impl.h

```diff
@@ -724,6 +728,17 @@ class DBImpl : public DB {
   static Status CreateAndNewDirectory(Env* env, const std::string& dirname,
                                       std::unique_ptr<Directory>* directory);
 
+  // Given a time window, return an iterator for accessing stats history
+  virtual Status GetStatsHistory(
```

Comment:
Nit: remove `virtual`. Use only `override`. Follow the updated, enforced code convention.

Reply:
Disregard this